### PR TITLE
Add checklist and responsibility document for Fireside chat

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

* Adds two subchapters, one with planning checklist, and other with the roles and responsibilities

### List of changes proposed in this PR (pull-request)

* Planning checklist issue template is linked to the landing page
* two subchapters with planning checklist and roles and responsibilities

### What should a reviewer concentrate their feedback on?

- [ ] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file:

### CODE REVIEW

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- Identify someone from the community who can do the introduction of The Turing Way, code of conduct reminder and pass it to the speakers
+
+## During the session
+
+- Open the Zoom call 15 minutes in advance (keep the waiting room of Zoom enabled)
+- test if speakers' microphone, camera and internet work alright - help troubleshoot any tech challenge
+- Let participants in right on time
+- Welcome them and share Etherpad
+- Remind them that the call will be recorded and that participants can use chat (but may not have the chance to speak during the 60 minutes live)
+- start recording and enable transcription
```
[the remark this answers is not in the document]
Comment (Arielle-Bennett) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:38]: Another thought, do we want the recording to be on someone's local computer or in the cloud?

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+- In 5 minutes past as people join, the person designated to welcome them will introduce the Turing Way, CoC, and Etherpad information and Present the topic
+- Hosts then introduce themselves and post the opening questions allowing all speakers including the co-host to share their position on the topic in 2 minutes
+- As planned in the agenda, hosts ask the question posed to the specific speaker, follow up response from the other speaker is invited
+- Speakers will make sure that everyone on the panel has had an equal chance to discuss and share their opinions
+- Any question from the chat is copied over the Etherpad, in the last 10 minutes publically posed question can be asked
+- The recorded session finish with 30 second closing argument
```
[the remark this answers is not in the document]
Comment (Arielle-Bennett) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:44]: Although, we may just want to limit this to speakers?

File: book/website/community-handbook/fireside-chat/fireside-chat-roles.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+* They will supply a short bio and image to be shared on the Eventbrite page.
+* They will be invited to attend a call (30) minutes before the event to discuss the topic and help the co-hosts identify what specific area of the topic they will talk about.
+* They will be coordinating with the co-host to finalise the specific question posed to them, and if they would like, they can suggest another question for the other panellists.
+* They will have access to the internal planning documents where they can find all information they need, as well as have opportunities to give feedback by directly collaborating on the Google document.
+* They should check what in addition to questions posed to them, questions they would like to respond to as a follow-up discussion.
+* Join Zoom at least 5-10 minutes before the event to test their system (microphone, camera, internet, and access to document used for the session)
+* It is important that all speakers get the same ‘stage time’ -- so please take this space to share your thoughts if you are generally not the one speaking out, and if you generally tend to speak more, please intentionally create space for others to speak.
```
Comment (Arielle-Bennett) [inline book/website/community-handbook/fireside-chat/fireside-chat-roles.md:60]: From here down the person changes from "they will" (third person) to "you" (second person) - not sure which one we prefer to use

**Review by Arielle-Bennett — APPROVED** [review]

**Comment by malvikasharan** [comment]
Brilliant! Thank you so much @Arielle-Bennett. Just in time for me to send the email to the next Fireside Chat collaborators. @aleesteele I won't merge this yet as you wanted to add your notes too. I will share the preview links to the folks for now.
Have a lovely evening! :trophy:

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:38]: Oh good point. Cloud is the best option.

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
+(ch-fireside-chat-checklist)=
+
+# Fireside Chat Planning Checklist
+
+## Before the Event
+
+- Create a planning issue on GitHub using the [issue template - Fireside Chat Checklist](https://github.com/alan-turing-institute/the-turing-way/issues/new/choose)
+- Create a Zoom link with a waiting room and live transcription enabled
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:8]: There is indeed no Turing Way zoom account. Either @aleesteele or I create from our personal account and invite speakers (if from the Turing) as co-hosts (we always have at least one Turing member on the panel). I can edit to reflect that.

[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:8]:
```suggestion
- A Turing Way team member (Community Manager or Research Project Manager) will create a Zoom link with a waiting room and live transcription enabled. They will also open the call on the day of the event, and give 'co-host rights' to all facilitators to allow them to manage Zoom participants and chats.
```

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md]: That's true. I have removed the time limit now.

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+(ch-fireside-chat-checklist)=
+
+# Fireside Chat Planning Checklist
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:3]: True, we do have a template planning document listed I believe, but that can definitely be moved to a subchapter and linked as a template.

File: .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md [hunk 6]
```diff
@@ -1,4 +1,18 @@
...
+
+---
+
+**The upcoming Fireside chat is on Date - Month - Year**
+
+Register here to attend: [add registration page link].
+
+## Before the Event -- at least 6-8 weeks in advance
+---
 name: "\u1F525 Fireside Chat Checklist"
```
Comment (Arielle-Bennett) [inline .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md:16]:
```suggestion

```

This looks like it's duplicated from above? Ignore if not!

File: .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md [hunk 7]
```diff
@@ -1,4 +1,18 @@
...
 about: Fireside Chat Planning Checklist
```
Comment (Arielle-Bennett) [inline .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md:17]:
```suggestion

```

This looks duplicated - ignore if not

File: .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md [hunk 8]
```diff
@@ -1,4 +1,18 @@
...
 title:
```
Comment (Arielle-Bennett) [inline .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md:18]:
```suggestion

```

This looks duplicated - ignore if not

**Review by Arielle-Bennett — APPROVED** [review]
looks good! We've used this for review training this morning, thank you :)

File: .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md (hunk 6 above)
Comment (malvikasharan) [reply .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md]: It was duplicated!

File: book/website/community-handbook/fireside-chat/fireside-chat-checklist.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+- Download the video from Zoom and edit the live transcription (proofread cc)
+- Upload the video on The Turing Way youtube - label and annotate well, and add the flyer as the video's front page
+- Share the published videos with the speakers
+- Summarise the session to add to The Turing Way -- invite someone from the community who could do that 
+- Promote the video via Slack, Newsletter, Twitter
+- Send a thank you email to the Eventbrite participants sharing the video and inviting any ideas and suggestions for improvement via a standard feedback form
+- Update the book chapter and templates if needed
+- Update the video link here and update the section for the upcoming Fireside chat
+- Add speakers to this GitHub repo using `all-contributions bot` for presentation
+- Close the planning issue as complete!
```
Comment (malvikasharan) [inline book/website/community-handbook/fireside-chat/fireside-chat-checklist.md:61]:
```suggestion
- Send a thank you email to the speakers (within 1 day)
- Archive session notes by copying them from Etherpad to the internal planning document
- Download the video from Zoom and edit the live transcription (proofread for accuracy) -- invite volunteer support as needed
- Upload the video on The Turing Way youtube - label and annotate well, and add the flyer as the video's front page
- Share the published videos with the speakers
- Summarise the session to add to The Turing Way -- invite someone from the community who could help with that 
- Promote the video via Slack, Newsletter, Twitter
- Send a thank you email to the Eventbrite participants sharing the video and inviting any ideas and suggestions for improvement via a standard feedback form
- Update the book chapter and templates if needed
- Add speakers to the contributors table on GitHub repo using `all-contributions bot` for presentation
- Close the planning issue as complete!
```

File: book/website/community-handbook/fireside-chat/fireside-chat-roles.md [hunk 10]
```diff
@@ -0,0 +1,90 @@
+(ch-fireside-chat-roles)=
+
+# Fireside Chat Roles and Responsibilities
+
+(ch-fireside-chat-roles-chair)=
+## Chair - Opens the Session
+
+One of *The Turing Way* team members will open the panel, welcoming the audience, introducing *The Turing Way* and the Fireside Chat, and setting the norm (CoC, Etherpad, chat, question process). They’ll thank the co-hosts and pass the ‘mic’ to them.
```
Comment (malvikasharan) [inline book/website/community-handbook/fireside-chat/fireside-chat-roles.md:8]:
```suggestion
One of *The Turing Way* team members will open the panel, welcoming the audience, introducing *The Turing Way* and the Fireside Chat, and setting the norm (Code of Conduct [CoC], Etherpad, chat, question process). They’ll thank the co-hosts and pass the ‘mic’ to them. 
```

File: book/website/community-handbook/fireside-chat/fireside-chat-roles.md [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+* Following this discussion, *The Turing Way* team will suggest a few titles for the event on Slack and invite discussion to finalise the speakers to be invited.
+* Once speakers and titles are finalised, they will make sure that there is someone (co-hosts or from *The Turing Way* Team) delegated to contact the speaker using the email draft.
+* They will help set up the Eventbrite, Etherpad, Zoom and Flyer for online sharing -- inviting feedback and edits from the co-hosts and speakers. If a speaker hasn’t shared their bio and image, they will follow up.
+* They will share the speaker release form via email before the event.
+* They will upload the video on YouTube:
+  * *The Turing Way* team will invite speakers to help review the uploaded transcription for correctness.
+* They will send a “thank you email” to all the attendees via Eventbrite sharing the link to the YouTube video.
+* They will thank the speakers, and make sure that the signed speaker’s release form has been obtained.
```
Comment (malvikasharan) [inline book/website/community-handbook/fireside-chat/fireside-chat-roles.md:77]:
```suggestion
* Invite the co-hosts at least 6 weeks in advance and set up an initial planning discussion with them. 
* The planning discussion is crucial for *The Turing Way* team and co-hosts to explore the topics and themes they want to speak about and the possible set of speakers they would like to invite from their network.
* Following this discussion, the co-hosts will propose a few titles for the event and nominate speakers (via Slack).
* Based on the previous contacts with the proposed speakers, one of the co-hosts or a *Turing Way* team member will email the speaker sharing initial details and proposed theme. 
*The Turing Way* Team they will make sure that there is someone delegated to contact the speaker using a template email draft. 
They will also follow up and set up neetings for discussions as needed.
* They will help set up the Eventbrite, Etherpad, Zoom and Flyer for online sharing -- inviting feedback and edits from the co-hosts and speakers. If a speaker hasn’t shared their bio and image, they will follow up.
* They will send calendar invites for the event with link for joining the call.
* On the day of event, a team member will open the call, welcome participants and monitor chat and Etherpad.
* They will share the speaker release form via email before the event.
* They will upload the video on YouTube:
  * *The Turing Way* team will invite speakers to help review the uploaded transcription for correctness.
* They will send a “thank you email” to all the attendees via Eventbrite sharing the link to the YouTube video.
* They will thank the speakers, and make sure that the signed speaker’s release form has been obtained. 
```

File: book/website/community-handbook/fireside-chat/fireside-chat-roles.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+(ch-fireside-chat-roles-agenda)=
+## Event Agenda
+
+Fireside Chats follow this rough agenda:
+
+* The co-hosts will introduce the topic and the panel. The invited co-host will also introduce their organisation/community as co-organisers.
+* Each speaker will give a 2-minute pitch covering their interests, experiences, concerns and insights related to the event's topic.
+* Session hosts will state the question and invite responses from 1-2 speakers (the questions can be agreed upon in advanced) - there might be follow-up question based on the response that we may not have discussed in advance
+* Hosts will make sure that all speakers have had the chance to speak (by responding to different questions) and express their thoughts or share resources from their work.
+* If time permits, we will take questions from the audience and speakers will be invited to address them. We will however invite attendees to use Zoom chat and Etherpad notes to share their thoughts.
```
Comment (malvikasharan) [inline book/website/community-handbook/fireside-chat/fireside-chat-roles.md:96]:
```suggestion
* If time permits, co-hosts will take questions from the audience and speakers will be invited to address them.
They will also invite attendees to use Zoom chat and Etherpad notes to share their thoughts.
```

File: .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md [hunk 13]
```diff
@@ -13,11 +13,12 @@ Register here to attend: [add registration page link].
...
+- [ ] Identify an overarching theme that we share with a different community -- Fireside Chat is intended to promote cross-community collaboration
+- [ ] Reach out to a community representative who can co-facilitate the event with a core team member in The Turing Way
 - [ ] Set up an internal shared document for discussions and notes (The Turing Way shared google doc, see [this template](https://docs.google.com/document/d/1X_NfRkkH6p47yRgpd6xlw8yrvo6jIsbF_mV0BinjcaQ/edit?usp=sharing))
-- [ ] discuss what the co-hosts might want to highlight at this fireside chat (note: these discussions don't necessarily need to identify a solution - but recognised shared themes, challenges, and spaces some research communities share)
+- [ ] Discuss what the co-hosts might want to highlight at this Fireside Chat (note: these discussions don't necessarily need to identify a solution - [ ] but should recognise shared themes, challenges, and spaces for research communities
 - [ ] Identify speakers from the community and invite them
```
Comment (aleesteele) [inline .github/ISSUE_TEMPLATE/FIRESIDE_CHAT_CHECKLIST.md:22]: - [ ] Request sign of release form

File: book/website/community-handbook/fireside-chat/fireside-chat-roles.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+* If they already know the suggested speaker, we will appreciate it if they can send the invitation and get confirmation using an [email draft](https://docs.google.com/document/d/12VoVexsXXBx25SnAU4mayPw5un-v936g9Ux61DLjGzo/edit).
+* Host 30 mins calls with the speakers before the event to identify what they would like to speak about.
+* Frame questions based on their discussions with the speakers, and invite speakers to help finalise the question (using Slack and the planning document).
+* Considering the flow of the discussion, decide on the order of the questions that will be asked.
+  * Each co-host should have equal opportunity to get the ‘stage time’
+  * They should agree on which questions they will pose to guide the discussion.
+  * They should pose 1 question to each speaker who takes a lead on responding. 1-2 follow-up responses can be invited from other speakers.
+    * Make sure that the speakers know in what order they will be speaking, and what questions will be asked.
+    * Coordinate via the Slack channel, and share any notes about the format via the shared document.
```
Comment (AlexandraAAJ) [inline book/website/community-handbook/fireside-chat/fireside-chat-roles.md:42]: What happens if the host panels don't use Slack? Should we asked them which communications tool they would like us to use for coordinations?

**Review by AlexandraAAJ — COMMENTED** [review]
It looks very good to me, thanks!

**Pull request merged**